src/cljs/chronograph_web/views.cljs

```diff
@@ -2,8 +2,13 @@
   (:require [re-frame.core :as rf]
             [chronograph-web.subscriptions :as subs]))
 
+(defn login-route []
+  (let [location (-> js/window .-location .-search)
+        client-type (-> location (js/URLSearchParams.) (.get "client-type"))]
+    (str "/auth/google/login?client-type=" (or client-type "web"))))
```

Comment:
Does the client-type belong to a whitelisted set? (ios, android, web etc.)?

Reply:
It's either `web` or `desktop`.